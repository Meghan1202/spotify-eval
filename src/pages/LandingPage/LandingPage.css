.Sync-Buton {
    width: 4.8rem;
    height: 1.7rem;
    background-color: #4c63aa;
    border-radius: 2rem;
    color: white;
    border: none;
    font-weight: bold;
}

.Landing-Page {
    align-self: center;
    justify-content: center;
}