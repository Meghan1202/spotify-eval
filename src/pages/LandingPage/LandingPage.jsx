import React from 'react';
import PropTypes from 'prop-types';
import { Link } from 'react-router-dom';
import './LandingPage.css';

const LandingPage = ({ getSongs }) => (
  <div className="Landing-Page">
    <p>
      :(
      {' '}
      <br />
      Seems a bit empty in here...
      <br />
    </p>
    <Link to="/allSongs" className="Sync-Buton" type="button" onClick={() => { getSongs(); }}>Sync</Link>
  </div>
);
LandingPage.propTypes = {
  getSongs: PropTypes.func.isRequired,
};

export default LandingPage;
