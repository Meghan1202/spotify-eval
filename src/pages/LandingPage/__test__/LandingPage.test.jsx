import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import LandingPage from '../LandingPage';

describe(LandingPage.name, () => {
  it('should match the snapshot', () => {
    const { container } = render(
      <BrowserRouter>
        <LandingPage getSongs={jest.fn()} />
      </BrowserRouter>,
    );
    expect(container).toMatchSnapshot();
  });
});
