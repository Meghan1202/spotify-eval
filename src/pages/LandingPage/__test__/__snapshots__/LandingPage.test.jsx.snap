// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage should match the snapshot 1`] = `
<div>
  <div
    class="Landing-Body"
  >
    <div
      class="Landing-Page"
    >
      <p>
        :(
         
        <br />
        Seems a bit empty in here...
        <br />
      </p>
      <a
        class="Sync-Buton"
        href="/allSongs"
      >
        Sync
      </a>
    </div>
  </div>
</div>
`;
